src/olympia/addons/templates/addons/replacement_addons.html

```diff
@@ -0,0 +1 @@
+{% extends "impala/base_shared.html" %}
```

Comment:
Might be nice to add some copy here just to say it's a placeholder or "coming soon" or something?